lib/api/controllers/documentController.js

```diff
@@ -227,39 +255,39 @@ function DocumentController (kuzzle) {
    * @param {Request} request
    * @returns {Promise<Object>}
    */
-  this.mUpdate = function documentMUpdate (request) {
-    return doMultipleWriteActions(kuzzle, request, 'update');
-  };
+  mUpdate(request) {
+    return doMultipleWriteActions(this.kuzzle, request, 'update');
+  }
 
   /**
    * Replace a document through the persistent layer. Throws an error if the document doesn't exist
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 99 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)